### ISSUE TITLE
Number of contigs - filtering by default?

### ISSUE THREAD

**Issue by chrstraub** [tseemann/nullarbor#235]
Hi again,

I've got a question regarding the discrepancy in number of contigs in contigs.fa and contigs.gff.

I did a comparison of SKESA vs SPAdes assembler and in terms of N50 and number of contigs SPAdes outperforms SKESA, according to the final report that is created. 

**SKESA**

```
Isolate | Contigs | bp | ok | Ns | gaps | min | avg | max | N50 | CDS
18AR1237 | 117 | 2114148 | 2114148 | 0 | 0 | 504 | 18069 | 152163 | 46626 | 2079
18GQA01 | 117 | 2120247 | 2120247 | 0 | 0 | 520 | 18121 | 108270 | 38933 | 2078
```
**SPAdes**
```
Isolate | Contigs | bp | ok | Ns | gaps | min | avg | max | N50 | CDS
18AR1237 | 85 | 2135149 | 2134671 | 478 | 0 | 543 | 25119 | 207692 | 60466 | 2095
18GQA01 | 85 | 2138493 | 2137714 | 779 | 0 | 508 | 25158 | 207737 | 52612 | 2100
```
But when I have a look at the raw assembly files, i.e. contigs.fa - they have a much higher number of contigs. i.e. for SPAdes:

```
Sample | N contigs contigs.fa | N contigs contigs.gff | 
18AR1237_S30 |  317 | 85 
18GQA01_S6 | 339 | 85 
```
So my questions are:
- SPAdes usually creates a scaffolds.fa file (as seen in log - its put in a temp folder), but its not in the isolate folder? 
- what explains the discrepancy in contig numbers -  the default behaviour must be that contigs are filtered based on length. So are sequences < x length of nucleotides removed? whats the cut-off? 500 bp? And is there a way to change that?
- or does prokka only report contigs where it has found annotations?

**Comment by tseemann** [comment]
The `--asembler` option controls which plugin script is run.
The plugins are:
```
ls nullarbor/plugins/assembler/*.sh
megahit.sh  shovill.sh  skesa_fast.sh  skesa.sh  spades.sh
```

**Comment by tseemann** [comment]
TLDR;
1.  `contigs.fa` _is_ spades' `scaffolds.fa`, it's renamed to be consistent
2. `MIN_CTG_LEN=500`  and use `make MIN_CTG_LEN=100` to change it
3. prokka includes all input contigs in its output, but `--minctglen` is set

**Comment by chrstraub** [comment]
Thanks Torsten, for clarifying this!
